.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         ghc: ["9.4.8", "9.6.6", "9.8.4"]
-        cabal: ["3.10.3.0"]
+        cabal: ["3.14.1.1"]
```

Comment:
If we update this here, then we ought to update it in the Dockerfile as well.

Reply:
Good point! Done!